### ISSUE TITLE
index.php is not using any php so it would make more sense to have to named as index.html

### ISSUE THREAD

**Issue by aidanok** [aliceasuna94/Arweave-MultiCheckURL#2]
Hey @aliceasuna94 

Since index.php is not actually using php and is deployed as html / js dApp, it would make more sense to name it index.html

**Comment by rawakinode** [comment]
Yes, true. I did make in PHP. But when I deployed, I changed to HTML. I will renew with changes.
